src/api/CompileTools.ts

```diff
@@ -710,12 +719,12 @@ export namespace CompileTools {
         }
       }
 
-      commandUI.addButtons({ id: `execute`, label: `Execute` });
+      commandUI.addButtons({ id: `execute`, label: `Execute` }, { id: `cancel`, label: `Cancel` });
 
       const page = await commandUI.loadPage(name);
```

Comment:
This will fix eslint's complain
```suggestion
      const page = await commandUI.loadPage<any>(name);
```